src/main/java/com/patsi/service/PersonInfoService.java

```diff
@@ -22,6 +23,8 @@ public class PersonInfoService {
     EmailService emailService;
     Logger log = LoggerFactory.getLogger(PersonInfoService.class);
 
+    @Value("${enableEmail}")
```

Comment:
group these flag toggling in to a separated service

Reply:
Fixed :D